React-Template-master/src/components/MessageModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Modal, Box, Typography, Button, useTheme } from '@mui/material';
+import { NEUTRAL } from '../theme/palette';
+import { CorrectLogo, WarningLogo } from './Logo';
+import { useTranslation } from 'react-i18next';
+
+interface ConfirmationModalProps {
+    open: boolean;
+    setOpen: Function;
```

Comment:
Don't mention the type as Function, it varies depends on the function, give a more specific type

Reply:
okay